src/components/ebay-fake-link/fake-link.stories.js

```diff
@@ -0,0 +1,95 @@
+import component from './index.marko';
+import readme from './README.md';
+
+const Template = (args) => ({
+    input: {
+        ...args,
+        renderBody(out) {
+            out.html(args.renderBody);
+        },
+    },
+});
+
+export default {
+    title: 'ebay-fake-link',
+    component,
+    parameters: {
+        docs: {
+            description: {
+                component: readme,
+            },
+        },
+    },
+    argTypes: {
+        renderBody: {},
+        href: {
+            description: 'for link',
+        },
+        disabled: {
+            description: '',
+            table: {
+                category: 'Toggles',
+                defaultValue: {
+                    summary: 'false',
+                },
+            },
+        },
+        'partially-disabled': {
```

Comment:
Let's remove this option from fake-links. Typically partially-disabled is only used on pagination and carousel paddles.